release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3248,6 +3248,15 @@ In the following tables, features are marked with the following statuses:
 
 * {product-title} version {product-version} disconnected installation fails on {ibm-power-name} Virtual Server. (link:https://issues.redhat.com/browse/OCPBUGS-36250[*OCPBUGS-36250*])
 
+* In {hcp} for {product-title}, if you disable the xref:../installing/cluster-capabilities.adoc#ingress-operator_cluster-capabilities[Ingress capability], the Console Operator returns the following error message:
++
+[source,text]
+----
+RouteHealthAvailable: failed to GET route.
+----
++
+To avoid this error, do not disable the ingress capability in an {product-title} managed cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33788[*OCPBUGS-33788*])
```

Comment:
```suggestion
To avoid this error, do not disable the Ingress capability in an {product-title} managed cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33788[*OCPBUGS-33788*])
```
I would cap it here, too, since this appears to be the name of the API.